test/operator/operator_test.py

```diff
@@ -643,7 +642,7 @@ def test_nonlinear_functional_operators():
 
     assert not C.is_linear
     assert almost_equal(C(x), A(x * 2.0))
-    
```

Comment:
In general such changes shouldnt be in Pull requests, I know we both do it, but we should try to avoid it. This causes some extra trubble in the review.